Felienne/hedy
# [FEATURE] Implemented explore page

**Description**
In this PR we implement an explore page. A way for users, old and new a-like to find interesting programs! They can filter on level and/or adventure within all shared programs to find interesting one to try, copy and alter. We implementation a new button in the menubar called `explore`. All public shared programs are automatically shown.

**We should merge #1629 first as we need this one for the new limit() function for db calls**

**Fix for**
This PR fixes #1575.

**How to test**
Notice the new option on the menubar. Navigate to this `explore` and try filtering on level, adventure or not at all. Select any program to open it in the view page, enabling you to run it!
